sql/mysqld.h

```diff
@@ -119,6 +119,10 @@ extern bool opt_ignore_builtin_innodb;
 extern my_bool opt_character_set_client_handshake;
 extern my_bool debug_assert_on_not_freed_memory;
 extern bool volatile abort_loop;
+extern my_bool redirect_enabled;
+extern const char *redirect_server_host;
+extern const char *redirect_server_port;
+extern const char *redirect_server_ttl;
```

Comment:
const char * for the ttl. It says, the time is in seconds. It should be a number, should not it?
the same question is about the port. Is it not numerical?

Reply:
Changes made.